CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+Contributing to RasCAL-
+==========================
+Everyone is welcome to contribute to the RasCAL2 project by either opening an issue (please check that the 
+issue has not been reported already) or submitting a pull request.
+
+Create Developer Environment
+----------------------------
+First begin by creating a fork of the RasCAL-2 repo, then clone the fork
+
+    git clone https://github.com/<username>/RasCAL-2.git
+    cd RasCAL-2
+
+We recommend using anaconda python distribution  for development, a new virtual environment should be 
+created to isolate dependencies. For Windows, first download and install [Microsoft Visual C++](https://aka.ms/vs/16/release/vc_redist.x64.exe). Then run the following
+
+    conda env create -f environment.yaml
+    conda activate rascal2
+
+And finally create a separate branch to begin work
+
+    git checkout -b new-feature
```

Comment:
```suggestion
    git switch -c new-feature
```
`git switch` is recommended over `git checkout` in [new Git tutorials](https://git-scm.com/docs/gittutorial) to reduce confusion over how much `checkout` does
[since git 2.23](https://github.com/git/git/blob/master/Documentation/RelNotes/2.23.0.txt)